audio-worklet/noisegate-test.html

```diff
@@ -33,6 +32,70 @@
 
       let audit = Audit.createTaskRunner();
       
+      // The first three tests verify that the noise gate's threshold matches
+      // the input power closely. Zero dBFS should be equivalent to a unit
+      // amplitude of 1, -6 dBFS should be equivalent to ~0.5, and -12 to ~0.25,
+      // etc. In general, the decibel level is 10 * log10(a^2), where a is the
+      // amplitude of the sine wave.
+      
+      audit.define('unit-amplitude-matches-0-db', (task, should) => {
+        // The threshold should be set slightly above the oscillator's gain, so
+        // that the noise gate mutes the signal.
+        const oscillatorGain = 1;
+        const threshold = 10 * Math.log10(oscillatorGain * oscillatorGain)
+            + 0.015;
+        
+        let context = setupThresholdTestGraph(threshold, oscillatorGain);
+
+        // The noise gate should never open since the envelope follower won't
+        // quite reach 0 dB.
```

Comment:
This comment is wrong in light of what we've seen and contradicts the threshold in line 45.